linuxbrew-build.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ABS_PATH_LIMIT=85
+export HOMEBREW_NO_AUTO_UPDATE=1
+
+[[ -x ./bin/brew ]] || (echo "This script should be run inside Linuxbrew directory."; exit 1)
+
+cd $(realpath .)
+BREW_HOME="$PWD"
```

Comment:
This, on the other hand, does not need quotes:
```suggestion
BREW_HOME=$PWD
```
Test:
```
$ X="a b"
$ Y=$X
$ echo $Y
a b
```